code/__DEFINES/reactions.dm

```diff
@@ -7,7 +7,7 @@
 #define PLASMA_MINIMUM_OXYGEN_NEEDED		2
 #define PLASMA_MINIMUM_OXYGEN_PLASMA_RATIO	30
 #define FIRE_CARBON_ENERGY_RELEASED			100000	//Amount of heat released per mole of burnt carbon into the tile
-#define FIRE_HYDROGEN_ENERGY_RELEASED		280000  //Amount of heat released per mole of burnt hydrogen and/or tritium(hydrogen isotope)
+#define FIRE_HYDROGEN_ENERGY_RELEASED		2800000  //Amount of heat released per mole of burnt hydrogen and/or tritium(hydrogen isotope)
```

Comment:
Was this change intentional?

Reply:
yes, i removed the *10 from the burned_fuel part of the code and moved it there to make it official.